.prettierrc

```diff
@@ -0,0 +1 @@
+printWidth: 80
```

Comment:
80 is the default so i dont think this is necessary 
![image](https://user-images.githubusercontent.com/120322/108404044-420bb100-71d4-11eb-9abd-ed9cfdf2625b.png)

https://prettier.io/docs/en/options.html

Reply:
I found that prettier seemed to ignore the presence of an empty file. :disappointed: